decision-process/README.md

```diff
@@ -0,0 +1,134 @@
+    - `obsolete` – The RFC or decision was replaced by something newer (which should be linked in the Markdown content) or is just not relevant anymore (example: product or technology not used anymore).
+  - `summary` should summarize the RFC in 1-3 concise sentences. It is meant for people who want to get a very quick overview instead of reading whole RFC documents. That is, _most people_! We use this in the rendered version to provide a simple overview.
+- YAML headers will be checked by CI upon creation or push of a PR in the [`rfc` repo](https://github.com/giantswarm/rfc)
+
+Other than that, an RFC only requires a heading and otherwise remains free-form.
+
+The following is an entire and valid start of an RFC markdown file:
+
+```text
+---
+creation_date: 2023-07-13
+issues:
+  - https://github.com/giantswarm/giantswarm/issues/24661
+owners:
+  - https://github.com/orgs/giantswarm/teams/team-horizon
+state: review
+---
+
+# This is the RFC title
+```
+
+## Overview of RFCs by adding a list to the handbook
+
+We previously lacked an overview of RFCs. There was no way to sort RFCs by date, see which one is still relevant, or get a one-sentence summary in order to quickly skim through the decisions Giant Swarm has made in the past.
+
+Therefore, we will automatically render the merged RFCs into our [handbook](https://handbook.giantswarm.io/docs/) and provide at least a table as overview.
+
+## Decision making process
+
+- **Author opens pull request**
+
+  - Clone the [`rfc` repo](https://github.com/giantswarm/rfc)
+  - Create a new directory such as `nginx-ingress-app-migration`. It should be named like the others, i.e. without spaces. Mind that this name (so-called _slug_) is later used in the handbook URL of the rendered RFC and should not change.
+  - In that directory, create a `README.md` with the following template:
+
+    ```text
+    ---
+    creation_date: YYYY-MM-DD # please fill this in
+    issues:
+      - https://github.com/giantswarm/giantswarm/issues/FILL_IN_YOUR_ISSUE
+    owners:
+      - https://github.com/orgs/giantswarm/teams/team-FILL_IN_YOUR_TEAM # or https://github.com/orgs/giantswarm/teams/sig-FILL_IN_YOUR_SIG
+    state: review
+    ---
+
+    # This is the RFC title
+    ```
+
+  - Replace the placeholders
+  - Push your branch and open a pull request
+- **Author asks their own team/SIG for a short review.** They should check if the content and grammar make sense. There should be no discussion yet about the decision since not all stakeholders are invited at this time.
+- **Review phase begins.** The author posts a link to the PR to appropriate `#news-*` or other Slack channels, and adds stakeholders as reviewers on the PR. The reviewers use GitHub comments on the PR. They should not push directly to the PR branch unless agreed with the author(s).
+- **Any employee can review. The relevant SIG takes care that the RFC progresses and does not become stale.**
```

Comment:
@stone-z commented that the RFC author should primarily drive the effort, in their own interest.

@stone-z / @puja108 reported previously using a "review until" date. We'll have to see how this works out culturally. Please comment if you want such a concept in here for the "hard" cases. We discussed in SIG Architecture that RFCs which are important strategically or for the roadmap will usually progress without problems.

Reply:
Mentioned author as primary driver